PurchasesTests/TestHelpers/ASN1ObjectIdentifierEncoder.swift

```diff
@@ -0,0 +1,65 @@
+//
+// Created by Andrés Boedo on 8/11/20.
+// Copyright (c) 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+@testable import Purchases
+
+class ASN1ObjectIdentifierEncoder {
```

Comment:
this ended up being tougher than the decoder. 
it's also less efficient since I didn't go crazy optimizing given that it's only for tests.

Reply:
I did manually test that it outputs the correct result for all known object identifiers.